### PR TITLE
Demo

### PR DESCRIPTION
**Pull request by SamirBoulil**
Introduce a small fix to improve the feedback loop in the team.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -118,3 +118,5 @@ The way you unpublish a PR from the reminder is by telling Slub in the following
 *In this example, the slub bot name is @Yeee*
 
 ![Unpublishing reminders](https://miro.medium.com/max/1760/1*lKYEB13fVb1lQzvL5pVtKg.png)
+
+Hello demo!
```
Comment (SamirBoulil) [inline Readme.md:122]: Hello

Comment (samirapptesting) [reply Readme.md:122]: I have a comment regarding this change!

**Review by samirapptesting — CHANGES_REQUESTED** [review]
No

**Review by samirapptesting — APPROVED** [review]
yes

**Pull request closed without merging**